### ISSUE TITLE
Enhancement: Windows installation is not indexed by search

### ISSUE THREAD

**Issue by Xammie** [jhen0409/react-native-debugger#360]
The windows installer of react native debugger installed in my AppData folder. Windows search automatically excludes this folder from the search index. Because of this i cannot open react native debugger from my windows search.

I solved this myself by adding a shortcut of react-native-debugger.exe in the programs folder wich is will be indexed (`C:\ProgramData\Microsoft\Windows\Start Menu\Programs`).

It would be nice if the installation wizard did this automatically! :)

React Native Debugger app version: 0.9.9
React Native version: 0.59.9
Operating System: Windows

**Comment by Legi0n1138** [comment]
Still having this issue on the latest version.
Could you please at least mention it somewhere in the docs? Took me a while to find the install location.

**Comment by cnmoise** [comment]
Ran into this same issue, would be nice if there was some info on the github https://github.com/jhen0409/react-native-debugger/issues/533

**Comment by Kif11** [comment]
I agree that was confusing to me as well. I would run the original executable every time I want to use the debugger. 😅 @Xammie solution worked great. I created shortcut form `%HOMEPATH%\AppData\Local\react_native_debugger\react-native-debugger.exe` to `C:\ProgramData\Microsoft\Windows\Start Menu\Programs` and now I can see it in my start menu search.

**Comment by Ahmdrza** [comment]
Same issue for me. I was using installer to open debugger every time, even search in the whole C: folder wasn't finding anything. Glad I found this comment, just sent a shortcut to the desktop.

**Comment by faltinyuva** [comment]
My  alternative for the same issue was  right click `%HOMEPATH%\AppData\Local\react_native_debugger\react-native-debugger.exe` and select "Pin to Start"
